functions.php

```diff
@@ -34,7 +34,7 @@
     );
 }
 
-if ($composer) {
+if (file_exists($composer)) {
```

Comment:
Yeah, that's why I said the solution was ugly. Just wanted to raise it as an issue